samples/drivers/led/led_strip/boards/adafruit_feather_esp32s3_procpu.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_POWER_DOMAIN=y
+CONFIG_GPIO=y
+CONFIG_PM_DEVICE=y
+CONFIG_PM_DEVICE_POWER_DOMAIN=y
```

Comment:
Is all this necessary to demonstrate the `led-strip` driver on this board ? For example, I can see that `CONFIG_GPIO` is already enabled in the board defconfig and as far I can tell it is not a dependency of the WS2812 LED strip driver...